lib/room.rb

```diff
@@ -1,12 +1,25 @@
-class Rom
-
+class Room
+  attr_reader :category, :length, :width
   def initialize(category, length, width)
     @category = category
-    @length = 10
+    @length = length
     @width = width
+    @coats_of_paint = 0
   end
 
   def area
-    @length * @width
+    @length.to_f * @width.to_f
+  end
+
+  def painted?
```

Comment:
Keep a close eye on the interaction pattern!  The method should be called `is_painted?`